packages/rocketchat-ui/client/views/app/room.js

```diff
@@ -721,6 +721,52 @@ Template.room.events({
 		}
 	},
 
+	'click .recognized-term'(e) {
+		if (!Meteor.userId()) {
+			return;
+		}
+		const term = $(e.currentTarget).get(0).innerText;
+
+		Meteor.setTimeout(() => {
+			if ($('.messages-container-wrapper .contextual-bar').length === 0) { // the tab bar is closed
+				// open the tab
+				$('div:not(.active) .rc-header .rc-room-actions .rc-room-actions__action[data-id="assistify-ai"] > button').click();
```

Comment:
Could there be more than one `.rc-room-actions__action[data-id="assistify-ai"] > button`? Else I would simplify the expression to contain only this part. Or maybe remove at least `.rc-header .rc-room-actions` if you want to make sure that the button only gets clicked if it is not in an `.active` div.

Reply:
made it more specific for performance reasons